javascript-methods/main.js

```diff
@@ -0,0 +1,58 @@
+var num1 = 5;
+var num2 = 15;
+var num3 = 7;
+
+var maximumValue = Math.max(num1, num2, num3);
+
+console.log('value of maximumValue:', maximumValue);
+
+var heroes = ['spiderman', 'baymax', 'mystique', 'the incredibles'];
+
+var randomNumber = Math.random();
+randomNumber += randomNumber * heroes.length;
```

Comment:
Make sure you're just reassigning the new `randomNumber`, not adding to the previous value:

---

![image](https://user-images.githubusercontent.com/16143771/187734821-2ae24985-0ac7-4619-87aa-1e83dc69465c.png)